### PR TITLE
dont wait anymore to emr status, just load the jobs

### PR DESCRIPTION
**Pull request by alonrolnik**
this is for managing multiple steps with the same cluster.

### CODE REVIEW

File: emr_run_spark.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def _get_step_ids_for_job_flow(job_flow_id, client):
 
 def _wait_for_job_flow(aws_region, job_flow_id, step_ids=[]):
     while True:
-        time.sleep(5)
+        time.sleep(30)
```
Comment (rantav) [inline emr_run_spark.py:248]: BTW, why did you change to 30 if you're not waiting for it at all?

**Comment by rantav** [comment]
lgtm, just a small question regarding - `time.sleep(30)`

File: emr_run_spark.py (hunk 1 above)
Comment (alonrolnik) [inline emr_run_spark.py:248]: I change it long time ago cause I noticed that Amazon throttled us in 5 seconds and it was not necessary to sample them every 5 seconds.
